pkg/products/amqonline/reconciler.go

```diff
@@ -350,3 +359,43 @@ func (r *Reconciler) reconcileBlackboxTargets(ctx context.Context, installation
 
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
+
+func (r *Reconciler) reconcilePrometheusRule(ctx context.Context, installation *integreatlyv1alpha1.Installation, client k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	monitoringConfig := config.NewMonitoring(config.ProductConfig{})
+	rule := &monitoringv1.PrometheusRule{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rhmi-amq-online-sli",
+			Namespace: r.Config.GetNamespace(),
+		},
+	}
+
+	rules := []monitoringv1.Rule{}
+	rules = append(rules, monitoringv1.Rule{
+		Alert: fmt.Sprintf("AMQ-SLI-1.1: AMQ Online console is not available in namespace %s", r.Config.GetNamespace()),
+		Annotations: map[string]string{
+			"sop_url": "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts_and_troubleshooting.md",
+			"message": "AMQ Online Console is not available",
+		},
+		Expr:   intstr.FromString(fmt.Sprintf("absent(kube_endpoint_address_available{endpoint=\"console\",namespace=\"%s\"}==1)", r.Config.GetNamespace())),
+		For:    "60s",
+		Labels: map[string]string{"severity": "critical"},
```

Comment:
Is this the right severity level? is the interval the right length?

Reply:
Severity level seems to be correct, it adheres to the logic presented in other alerts whereby the level is _critical_ if a service/container is down or unavailable and warnings are reserved for situations that aren't an immediate impact, so we should probably keep it the same for the alerts being created here aswell. As for the interval It seems to be also fine, especially when comparing it to the UPS console where the interval is much higher 
![image](https://user-images.githubusercontent.com/35736504/74171404-6f6b6980-4c26-11ea-94a1-eec0b615a274.png)